.github/workflows/wheels.yml

```diff
@@ -0,0 +1,60 @@
+name: Python wheels
+
+on:
+  pull_request:
```

Comment:
Any specific reason to build wheels on PR instead of only when pushing tags?

Reply:
I thought it's good to test in a pr, in case changes from pr prevent it from building.

do you want to change to only tags then?

Reply:
Yes, let's change it to tag-only please.

Reply:
ecd7c5f5151242c9dc1a43af8918b9d71aa52158

Reply:
also found original question i asked about this:
- https://github.com/libfuse/python-fuse/pull/66#issuecomment-1950229832